CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ This repo uses [Jinja2](https://www.palletsprojects.com/p/jinja/) to generate Do
 
 Make sure to have the following dependencies installed to compile the Dockerfiles.
 
-- Python 3.6+
+- Python 3.7+
```

Comment:
This Python version should stay as `3.6+`. This indicates the version of Python that is required to run `generate_dockerfiles.py`. It's independent of the Python/CUDA versions used for the images themselves.